docs/v3/how-to-guides/automations/use-custom-event-grammar.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: How to emit and use custom events
```

Comment:
Proposal to keep this scoped to "emitting custom events"; "using" is covered by automations and triggers

Reply:
hmm i think the most conceptually interesting / useful part of this how to use `for_each` and `after` in the trigger definition in conjunction with custom events, so I would prefer to keep the trigger definition here since the How To is not that long as is - what do you think?

alternatively I _could_ split it out so that we had two how tos like "emitting custom events" and "using for_each and after" but "emitting custom event grammar" isnt sufficiently motivated without showing its use in the trigger / simulated events

Reply:
that's fair, i retract my proposal. No need to split this up right now, main goal is to get the organization in place because then editing will be easy.

The title does need to change though to remove "How to"

Reply:
updated in [4de2982](https://github.com/PrefectHQ/prefect/pull/18034/commits/4de2982ebc008bb795fe97fc1ba3128d29ecff36)